scripts/init.sh

```diff
@@ -9,8 +9,4 @@ if [ -z $CI_PROJECT_NAME ] ; then
    rustup update stable
 fi
 
-rustup target add wasm32-unknown-unknown --toolchain nightly
-rustup component add rust-src --toolchain nightly
-rustup target add wasm32-unknown-unknown --toolchain stable
-rustup install nightly-2020-10-06
-rustup target add wasm32-unknown-unknown --toolchain nightly-2020-10-06
+rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
Why do we still have a `nightly` here?

Reply:
Actually we use nightly latest, stable version while build. Polkadot reference [link](https://github.com/paritytech/polkadot/blob/master/scripts/init.sh). Line 12.

Reply:
@huythanhcere 
The latest rust stable version for now is [1.50.0](https://github.com/rust-lang/rust/releases). If we don't have any feature blockers it should be used.

Reply:
i removed nightly, checking build. If build successfully, i will commit.

Reply:
@AndreiNavoichyk if remove nighty, there is error ([url](https://drive.google.com/file/d/1diqqqNnvXURRulXWGZLAt7FGIs3_Syrx/view?usp=sharing))

Reply:
I think you should approve this PR. There is not any improvement that we can do.

Reply:
@huythanhcere 
We should use `stable` toolchain here. Any reasons why not?
`rustup target add wasm32-unknown-unknown --toolchain stable`

Reply:
Error, details here: https://github.com/Cerebellum-Network/pos-network-node/runs/2191567050?check_suite_focus=true @AndreiNavoichyk

Reply:
So we depend on `nightly` version of `wasm32-unknown-unknown`.
Ok, I've refactored a little bit the structure. Let's wait for build.

Reply:
Thanks